app/models/concerns/users/callback_dashboard.rb

```diff
@@ -0,0 +1,50 @@
+module Users::CallbackDashboard
```

Comment:
@hengsokly please consider to rename Users::CallbackDashboard to Users::GrafanaConcern and class name Dashboard to Grafana